tests-invoke

```diff
@@ -107,6 +107,12 @@ def main():
                 body += f", [logs]({logs_url}/log.html)"
             if test_scenario != "":
                 title += f"/{test_scenario}"
+
+            assert test_os is not None
+            rel = test_os.split('-')[-1]
```

Comment:
This should be a bit stronger and check if it starts with `fedora-`. Also, let's check for `updates-testing` first, so that none of this runs for other scenarios.

Reply:
Correct, adjusted it all.